code/datums/shuttles.dm

```diff
@@ -227,10 +227,16 @@
 /datum/map_template/shuttle/emergency/scrapheap
 	suffix = "scrapheap"
 	name = "Standby Evacuation Vessel \"Scrapheap Challenge\""
-	credit_cost = -1000
+	credit_cost = 1000
 	description = "Due to a lack of functional emergency shuttles, we bought this second hand from a scrapyard and pressed it into service. Please do not lean to heavily on the exterior windows, they are fragile. <span class='danger'>VIRUS ALERT: This entry seems to be booby-trapped with a redirector trap. Buy this at your own risk.</span>"
 	admin_notes = "An abomination with no functional medbay, sections missing, and some very fragile windows. Surprisingly airtight."
 
+/datum/map_template/shuttle/emergency/scrapheap/prerequisites_met(mob/user)
+	if("emagged" in SSshuttle.shuttle_purchase_requirements_met)
```

Comment:
You only got half the code here, so this check doesn't actually do anything.

You need to add the `SSshuttle.shuttle_purchase_requirements_met |= "emagged"` to the `computer/communications.dm` in order to get this to work properly.

https://github.com/Citadel-Station-13/Citadel-Station-13/blob/d75a9cf36815aed3ed5519f1a1b3b7383213fa46/code/game/machinery/computer/communications.dm#L420-L427

Reply:
Thank you for the help! I did have a changes to see that its not locked away. Thought we already had a Syda shuttle that was emag only. Guess its admin made/spawned in